packages/e2e-test-utils-playwright/src/request-utils/global-styles.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Internal dependencies
+ */
+import type { RequestUtils } from './index';
+
+// https://developer.wordpress.org/rest-api/reference/themes/#definition
+async function getCurrentThemeGlobalStylesPostId( this: RequestUtils ) {
+	type ThemeItem = {
+		stylesheet: string;
+		status: string;
+		_links: { 'wp:user-global-styles': { href: string }[] };
+	};
+	const themes = await this.rest< ThemeItem[] >( {
+		path: '/wp/v2/themes',
+	} );
+	let themeGlobalStylesId: string = '';
+	if ( themes && themes.length ) {
+		const currentTheme: ThemeItem | undefined = themes.find(
+			( { status } ) => status === 'active'
+		);
+
+		const globalStylesURL =
+			currentTheme?._links?.[ 'wp:user-global-styles' ]?.[ 0 ]?.href;
+		if ( globalStylesURL ) {
+			themeGlobalStylesId = globalStylesURL?.split(
+				'rest_route=/wp/v2/global-styles/'
+			)[ 1 ];
+		}
+	}
+	return themeGlobalStylesId;
+}
+
+async function updateGlobalStyles( this: RequestUtils, styles: Object ) {
```

Comment:
I need to add doc blocks.

Hopefully I don't have to add types for the global styles object 🙏 

If there's a type somewhere else that can be imported that would be nice.

Reply:
I checked in core-data, and it's mostly using `any` or `Record< string, Object >` as return values for selectors 😄 . 

I did create an interface for [GlobalStylesRevisions](https://github.com/WordPress/gutenberg/blob/8eb63ddb51525164860e785900d7d86544f37afc/packages/core-data/src/entity-types/global-styles-revision.ts), I guess it would also be helpful to have one for global styles.

This comment is zero helpful I know.

Reply:
Thanks, I didn't know those existed. Could be useful for the utils if we can import the types from there. I'd feel better about adding a global styles type in that location.